src/components/DonationForm/DonationForm.jsx

```diff
@@ -1,33 +1,43 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
-import {
-  FormLabel,
-  Input,
-  InputGroup,
-  Select,
-  FormControl,
-  FormErrorMessage,
-  Button,
-} from '@chakra-ui/react';
+import { FormLabel, Input, Select, FormControl, FormErrorMessage, Button } from '@chakra-ui/react';
 import { yupResolver } from '@hookform/resolvers/yup';
 import { useForm, useFieldArray } from 'react-hook-form';
 import * as yup from 'yup';
 import styles from './DonationForm.module.css';
 import FurnitureField from '../FurnitureField/FurnitureField';
+import { sendEmail } from '../../utils/utils';
+import dconfirmemailtemplate from '../EmailTemplates/dconfirmemailtemplate';
 
 const schema = yup.object({
   firstName: yup.string().required('Invalid first name'),
-  lastName: yup.string().required('Invalid first name'),
-  zipcode: yup.number().typeError('ZIP code must be a number'),
+  lastName: yup.string().required('Invalid last name'),
+  phoneNumber: yup
+    .string()
+    .required('Phone number is required')
+    .matches(/[0-9]{3}-[0-9]{3}-[0-9]{4}/, 'Phone number must match xxx-xxx-xxxx format'),
+  zipcode: yup
+    .string()
+    .required('ZIP Code is required')
+    .matches(/^[0-9]+$/, 'ZIP Code must be a number')
+    .min(5, 'ZIP Code must be exactly 5 digits')
+    .max(5, 'ZIP Code must be exactly 5 digits'),
   email1: yup.string().email('Invalid email').required('Email required'),
-  email2: yup.string().email('Invalid email').required('Email required'),
+  email2: yup
+    .string()
+    .required('Email required')
+    .test(
+      'matches-email1',
+      'Email fields do not match',
+      (value, ctx) => value === ctx.parent.email1,
+    ),
```

Comment:
add validation to check if default selections for `FurnitureField` are selected (as of now the default selections have a value of `''`). take a look at [this](https://stackoverflow.com/questions/66266929/how-to-use-yup-validation-on-dynamic-form-using-react-hook-form-with-usefieldarr) for inspiration